asreview/webapp/api.py

```diff
@@ -797,6 +797,29 @@ def export_project(project_id):
     )
 
 
+@bp.route('/project/<project_id>/finish', methods=["GET"])
+def api_finish_project(project_id):
+    """Mark a project as finished or not"""
+
+    # read the file with project info
+    with open(get_project_file_path(project_id), "r") as fp:
+        project_info = json.load(fp)
+
+    try:
+        project_info["projectFinished"] = not project_info["projectFinished"]
```

Comment:
Let's rename this into `reviewFinished`. In the future, we might want to extend the scope of the application.